toolkit/script_to_check_cli_outputs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 <current_version> <new_version>"
+  echo "Usage: $0 v2.6.0 v2.7.0"
+  exit 1
+fi
+SPIN_CURRENT=$1
+SPIN_LATEST=$2
+mkdir ~/$SPIN_CURRENT
+mkdir ~/$SPIN_LATEST
+cp -rp check_cli_output.py ~/$SPIN_CURRENT
+cp -rp check_cli_output.py ~/$SPIN_LATEST
+cd ~
```

Comment:
What do you think about using `/tmp`(or even better, `mktemp`) instead of `~`/`$HOME`?

Reply:
Thanks @vdice - great idea. Using `mktemp` now.